### PR TITLE
updating CoC link and security link

### PR DESCRIPTION
**Pull request by vsoch**
Signed-off-by: Vanessa Sochat <vsochat@stanford.edu>

https://github.com/opencontainers/org/issues/3

### CODE REVIEW

**Comment by vbatts** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -86,19 +86,17 @@ New additions will be met with skepticism. If you think there is a
...
-request on GitHub. Instead, disclose the issue responsibly by sending an email
-to security@opencontainers.org (which is inhabited only by the maintainers of
-the various OCI projects).
+If you find an issue, please follow the [security][security] protocol to report it.
 
 # Copyright and license
 
 Copyright © 2016 Docker, Inc. All rights reserved, except as follows. Code is released under the [Apache 2.0 license](LICENSE). This `README.md` file and the [`CONTRIBUTING.md`](CONTRIBUTING.md) file are licensed under the Creative Commons Attribution 4.0 International License under the terms and conditions set forth in the file [`LICENSE.docs`](LICENSE.docs). You may obtain a duplicate copy of the same license, titled CC BY-SA 4.0, at http://creativecommons.org/licenses/by-sa/4.0/.
+
+[security]: https://github.com/opencontainers/org/blob/master/security
```
Comment (dmcgowan) [inline README.md]: This looks like it needs a `.md`, the link 404s for me

File: README.md [hunk 2]
```diff
@@ -86,19 +86,17 @@ New additions will be met with skepticism. If you think there is a
...
+[code-of-conduct]: https://github.com/opencontainers/org/blob/master/.github/CODE_OF_CONDUCT.md
```
Comment (dmcgowan) [inline README.md]: This also 404s, maybe `https://github.com/opencontainers/org/blob/master/code-of-conduct.md`?

File: README.md (hunk 1 above)
Comment (vbatts) [reply README.md]: that change is in-flight

File: README.md (hunk 2 above)
Comment (vbatts) [reply README.md]: she has a PR up for this

File: README.md (hunk 1 above)
Comment (vsoch) [reply README.md]: The security link above will be to a folder, so it should work without the .md.

Comment (dmcgowan) [reply README.md]: Is there a PR somewhere to drop the `.md` from the file?

**Comment by vbatts** [comment]
On 26/02/19 11:09 -0800, Derek McGowan wrote:
>dmcgowan commented on this pull request.
>
>
>
>>
> # Copyright and license
>
> Copyright © 2016 Docker, Inc. All rights reserved, except as follows. Code is released under the [Apache 2.0 license](LICENSE). This `README.md` file and the [`CONTRIBUTING.md`](CONTRIBUTING.md) file are licensed under the Creative Commons Attribution 4.0 International License under the terms and conditions set forth in the file [`LICENSE.docs`](LICENSE.docs). You may obtain a duplicate copy of the same license, titled CC BY-SA 4.0, at http://creativecommons.org/licenses/by-sa/4.0/.
>+
>+[security]: https://github.com/opencontainers/org/blob/master/security
>
>Is there a PR somewhere to drop the `.md` from the file?

not yet. I'm not sure if she was _about_ to add that to the Contribution
PR

**Comment by dmcgowan** [comment]
Sounds good, once that PR is open and matches then looks good here. We should probably merge this at the same time as that PR.

**Comment by vsoch** [comment]
here you go @dmcgowan https://github.com/opencontainers/org/pull/6
Agreed! If we click fast enough, maybe we won't be in the catch-22 :)
When the PR in the org repo is good we should merge that first, and then double check the links here just in case there is an issue.

**Comment by vbatts** [comment]
LGTM

**Comment by vsoch** [comment]
The commits have been _squashed_!

**Comment by vsoch** [comment]
Code of conduct link is updated to be outside of .github (and in all caps) to reflect discussion in OCI call.

**Comment by dmcgowan** [comment]
LGTM

**Comment by vbatts** [comment]
LGTM

**Comment by vbatts** [comment]
lol

**Comment by vsoch** [comment]
I know, he's huge and terrifying. I think that makes it better?

**Pull request merged**